darkmoon/api/v1/metadata/schema.py

```diff
@@ -32,27 +32,27 @@ class Hashes(BaseModel):
 class HeaderInfo(BaseModel):
     """Holds HeaderInfo class properties for .exe files."""
 
-    machine_type: str = Field(
+    machine_type: str | None = Field(
```

Comment:
Please revert all these optional fields and instead refactor the models to such that they aren't needed. Maybe we need more than 1 model to handle some edge case instead of trying to repurpose a single model for multiple variants?